Tests/Marketplace/marketplace_services.py

```diff
@@ -1140,8 +1140,43 @@ def copy_and_upload_to_storage(self, production_bucket, build_bucket, successful
                     self._aggregation_str = agg_str
             logging.success(f"Uploaded {self._pack_name} pack to {prod_pack_zip_path} path.")
 
+        # handle dependenices zip upload when found in build bucket
+        self.copy_and_upload_dependencies_zip_to_storage(
```

Comment:
This should be in a condition since some uploads will not have those zipped dependencies (such as marketplacev2)